Src/Prt/Core/PrtValues.c

```diff
@@ -813,6 +829,11 @@ void PRT_CALL_CONV PrtMapRemove(_Inout_ PRT_VALUE *map, _In_ PRT_VALUE *key)
 	}
 }
 
+PRT_VALUE ** PRT_CALL_CONV PrtMapGetLValue( _Inout_ PRT_VALUE *map, _In_ PRT_VALUE *key, _In_ PRT_BOOLEAN cloneKey)
+{
+	return PrtMapUpdateHelper(map, key, cloneKey, PrtMkNullValue(), PRT_FALSE);
```

Comment:
The `PrtMkNullValue()` call here will leak memory when the key already exists. Maybe you mean `NULL` here?